docs/usage/configuration.md

```diff
@@ -76,6 +76,8 @@ The branches on which releases should happen. By default **semantic-release** wi
 
 **Note**: Once **semantic-release** is configured, any user with the permission to push commits on one of those branches will be able to publish a release. It is recommended to protect those branches, for example with [GitHub protected branches](https://help.github.com/articles/about-protected-branches).
 
+**Note**: The prerelease can't be named with `/`. For example `{ "name": "debug/*", "prerelease": true }` won't work. You have to manually remove `/` (example: `{ "name": "debug/*", "prerelease": '${name.replace(/\//, "-")}' }`)
+
```

Comment:
I wonder if this is rather a bug that could be fixed in code instead of adding a note about it?

Reply:
@gr2m I am wondering you're probably true. [Git allows slash in tags](https://stackoverflow.com/a/26382358/10826693). I'll try it fix it in another branch. I'd close this proposal.

Reply:
It's not a bug it's a feature. 🤣